packages/docgen/package.json

```diff
@@ -0,0 +1,44 @@
+{
+	"name": "docgen",
+	"version": "1.0.0-beta.0",
+	"description": "Autogenerate public API documentation from exports and JSDoc comments.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"jsdoc",
```

Comment:
We always put `wordpress` keyword in this array.

Reply:
b08b8d3cf3824addaac12c546316c909e7a38194